### DISCUSSION TITLE
Presets issue for **letterSpacing**

### DISCUSSION THREAD

**Discussion by EhsanKhalili** [tailwindlabs/tailwindcss#14307] [Help]
Hi, I have an issue about presets of letterSpacing:

**What version of Tailwind CSS are you using?**

"tailwindcss": "^3.4.10",
"prettier-plugin-tailwindcss": "^0.6.6"

**What build tool (or framework if it abstracts the build tool) are you using?**

"next": "^14.2.7"

**What version of Node.js are you using?**

v20.17.0

**What browser are you using?**

Chrome Version 128.0.6613.114 (Official Build) (64-bit)

**What operating system are you using?**

Windows

**Describe your issue**

I have a <a href="https://tailwindcss.com/docs/presets">Reusable Configuration Preset</a> like below:
```js
// @persiabme/toolkit/dist/styles/tailwind.config.js

/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./src/**/*.{html,jsx,tsx,ts}"],
  theme: {
    extend: {
       ...
    },
    fontSize: {
      xs: ["var(--font-size-xs)", "var(--line-height-xs)"],
      sm: ["var(--font-size-sm)", "var(--line-height-sm)"],
      base: ["var(--font-size-base)", "var(--line-height-base)"],
      lg: ["var(--font-size-lg)", "var(--line-height-lg)"],
      xl: ["var(--font-size-xl)", "var(--line-height-xl)"],
      "2xl": ["var(--font-size-2xl)", "var(--line-height-2xl)"],
      "3xl": ["var(--font-size-3xl)", "var(--line-height-3xl)"],
      "4xl": ["var(--font-size-4xl)", "var(--line-height-4xl)"],
      "5xl": ["var(--font-size-5xl)", "var(--line-height-5xl)"],
      "6xl": ["var(--font-size-6xl)", "var(--line-height-6xl)"],
      "7xl": ["var(--font-size-7xl)", "var(--line-height-7xl)"],
      "8xl": ["var(--font-size-8xl)", "var(--line-height-8xl)"],
      "9xl": ["var(--font-size-9xl)", "var(--line-height-9xl)"],
    },
    letterSpacing: {
      tightest: "var(--tightest)",
      tighter: "var(--tighter)",
      tight: "var(--tight)",
      normal: "var(--normal)",
      wide: "var(--wide)",
      wider: "var(--wider)",
      "more-wider": "var(--more-wider)",
      widest: "var(--widest)",
    },
  },
};
```

But when I want to use it in my code like below

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  presets: [require("@persiabme/toolkit/dist/styles/tailwind.config.js")],
  prefix: "",
  corePlugins: {
    preflight: false,
  },
};
```

Everything work fine:

<img width="857" alt="image" src="https://github.com/user-attachments/assets/c18fe77c-2f50-4240-8983-37db4fb1c071">

except **letterSpacing**, 

<img width="857" alt="image" src="https://github.com/user-attachments/assets/03b756b6-3797-44c6-8944-e0554edf9af4">

It seems the presets doesn't work for **letterSpacing**.

Thank you very much.

**Comment by thecrypticace** [comment]
Hey, I'm not setting this issue here:

<img width="511" alt="Screenshot 2024-08-30 at 13 39 44" src="https://github.com/user-attachments/assets/37f109f5-b036-4c46-ad51-9ff0f74fa255">

Does this issue go away if you reload VSCode? If not can you provide a project which you see this issue reproduce reliably?
